source/_integrations/weatherflow.markdown

```diff
@@ -0,0 +1,58 @@
+---
+title: "WeatherFlow"
+description: "Instructions on how to integrate your WeatherFlow tempest into Home Assistant."
+ha_release: "2023.10"
+ha_category: 
+  - Sensor
+  - Environment
+ha_platforms:
+  - sensor
+ha_iot_class: "Local Push"
+ha_config_flow: true
+ha_codeowners:
+  - '@natekspencer'
+  - '@jeeftor'
+ha_domain: weatherflow
+---
+
+This integration is a local-only WeatherFlow integration that reads weather data from all [WeatherFlow Tempest](https://weatherflow.com/tempest-weather-system/) compatible weather station on the local network.
+
+<div class='note'>
+It possible you may see slight deviations between the values reported in Home Assistant and the values in the WeatherFlow App. This is because the WeatherFlow app takes into account both forecasts and neighboring weather stations in addition to the local data which is used in this integration.
```

Comment:
```suggestion
You may see slight deviations between the values reported in Home Assistant and the values in the WeatherFlow App. This is because the WeatherFlow app considers both forecasts and neighboring weather stations in addition to the local data used in this integration.
```